ocaml/idl/datamodel_pool.ml

```diff
@@ -1024,6 +1034,30 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let set_telemetry_next_schedule =
+  call ~name:"set_telemetry_next_schedule" ~lifecycle:[]
+    ~doc:
+      "Set the time after when the next round of telemetry collection will be \
+       carried out"
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; ( DateTime
+        , "value"
+        , "The time after when the next round of telemetry collection will be \
```

Comment:
Is this a date or a time span? "The time after" makes it sound this is a timespan - like days or hours. But `DateTime` suggests this is a date. This needs to be clarified.

Reply:
this is a datetime / timestamp

Reply:
I updated the the `time` to `datetime`.